test/TestUtilities/Test.Utility/Signing/TrustedTestCert.cs

```diff
@@ -60,10 +63,18 @@ public class TrustedTestCert<T> : IDisposable
                 throw new InvalidOperationException($"The certificate used is valid for more than {maximumValidityPeriod}.");
             }
 #endif
-            StoreName = storeName;
-            StoreLocation = storeLocation;
-            AddCertificateToStore();
+            if (RuntimeEnvironmentHelper.IsMacOSX)
+            {
+                AddTrustedCert();
+            }
+            else
+            {
+                StoreName = storeName;
+                StoreLocation = storeLocation;
+                AddCertificateToStore();  
+            }
             ExportCrl();
+
         }
```

Comment:
Blank line after } not before.
```C#
    }

    ExportCrl();
}
```

Reply:
Fixed. Thanks.

Reply:
Doesn't look like it in latest commit.

Reply:
Thanks. Fixed. Not sure why it's not updated here. Just checked the code here: https://github.com/NuGet/NuGet.Client/blob/dev-hengliu-xplat-2implementAddCertOnMacOS/test/TestUtilities/Test.Utility/Signing/TrustedTestCert.cs#L78-L79

Reply:
Not fixed.  Please reread my initial comment.  You removed a blank line, but you also need to add a blank line before `ExportCrl();`.